### ISSUE TITLE
Filling the Rectange with The Graphics Library Not Filling the Whole Screen

### ISSUE THREAD

**Issue by BenDaviesGFX** [BenDaviesGFX/Chaos#2]
Example Code
```Java
    @Override
    public void render(Graphics g) {
        g.setColor(Color.RED);
        g.fillRect(0,0,1280,720);
    }
```

Error:
http://puu.sh/r4riX/a6429fa327.jpg

**Comment by BenDaviesGFX** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;y9HTdV0g&#x2F;7-fill-rect-is-not-filling-the-whole-canvas"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> Fill rect is not filling the whole Canvas</a>

**Comment by BenDaviesGFX** [comment]
This is fixed in commit 250ba2a2179e1b80f956f0aad1a4ff2edc1ebf3c
